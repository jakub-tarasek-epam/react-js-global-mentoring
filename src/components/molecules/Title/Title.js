import React from "react";
import { Header } from "./Title.styles";
import { Link } from "react-router-dom";

const Title = ({ isHeader }) => {
  return <Header isHeader={isHeader}>NetflixRoulette</Header>;
};

export default Title;
