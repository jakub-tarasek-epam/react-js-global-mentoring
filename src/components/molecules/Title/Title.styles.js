import styled from "styled-components";
import {colorsPallete, fontSize} from "theme/variables";

export const Header = styled.h1`
    color: ${colorsPallete.red};
    font-weight: 900;
    font-size: ${props => props.isHeader ? fontSize.xl : fontSize.xs};
    margin: 0;
`;