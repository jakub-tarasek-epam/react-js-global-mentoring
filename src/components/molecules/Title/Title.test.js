import React from 'react';
import renderer from 'react-test-renderer';

import Title from './Title';

describe('Movie list', () => {

  it('renders correctly title component', () => {
    const tree = renderer.create(<Title />).toJSON();
    expect(tree).toMatchSnapshot();
  });

});
