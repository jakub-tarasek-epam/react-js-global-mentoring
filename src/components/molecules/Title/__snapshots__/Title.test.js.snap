// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Movie list renders correctly title component 1`] = `
<h1
  className="sc-bdvvtL fwSkZz"
>
  NetflixRoulette
</h1>
`;
